templates/resources/metal_connection.md.tmpl

```diff
@@ -16,45 +16,33 @@ Use this resource to request the creation an Interconnection asset to connect wi
 
 ### Shared Connection for organizations without Connection Services Token feature enabled
 
-{{tffile "examples/resources/metal_connection/example_3.tf"}}
+{{tffile "examples/resources/metal_connection/shared_no_token_metal_to_fabric.tf"}}
 
 ### Shared Connection with authorization_code Non-redundant NIMF connection from Equinix Metal to a Cloud Service Provider via Equinix Fabric Port
 
-{{tffile "examples/resources/metal_connection/example_4.tf"}}
+{{tffile "examples/resources/metal_connection/shared_nimf_to_csp.tf"}}
 
 ### Shared Connection with authorization_code Non-redundant NIMF connection from Equinix Fabric Cloud Router to Equinix Metal
 
-{{tffile "examples/resources/metal_connection/example_5.tf"}}
+{{tffile "examples/resources/metal_connection/shared_nimf_from_fcr.tf"}}
 
 ## Argument Reference
 
 The following arguments are supported:
 
-* `name` - (Required) Name of the connection resource
-* `metro` - (Optional) Metro where the connection will be created.
-* `facility` - (**Deprecated**) Facility where the connection will be created. Use metro instead; read the [facility to metro migration guide](https://registry.terraform.io/providers/equinix/equinix/latest/docs/guides/migration_guide_facilities_to_metros_devices)
-* `redundancy` - (Required) Connection redundancy - redundant or primary.
-* `type` - (Required) Connection type - dedicated or shared.
-* `contact_email` - (Optional) The preferred email used for communication and notifications about the Equinix Fabric interconnection. Required when using a Project API key. Optional and defaults to the primary user email address when using a User API key.
-* `project_id` - (Optional) ID of the project where the connection is scoped to, must be set for.
-* `speed` - (Required) Connection speed - Values must be in the format '<number>Mbps' or '<number>Gpbs', for example '100Mbps' or '50Gbps'. Actual supported values will depend on the connection type and whether the connection uses VLANs or VRF.
-* `description` - (Optional) Description for the connection resource.
-* `mode` - (Optional) Mode for connections in IBX facilities with the dedicated type - standard or tunnel. Default is standard.
-* `tags` - (Optional) String list of tags.
-* `vlans` - (Optional) Only used with shared connection. Vlans to attach. Pass one vlan for Primary/Single connection and two vlans for Redundant connection.
-* `service_token_type` - (Optional) Only used with shared connection. Type of service token to use for the connection, a_side or z_side. (**NOTE: To support the legacy non-automated way to create connections, terraform will not check if `service_token_type` is specified. If your organization already has `service_token_type` enabled, be sure to specify it or the connection will return a legacy connection token instead of a service token**)
+{{ .SchemaMarkdown | trimspace }}
 
 ## Attributes Reference
```

Comment:
This `## Attributes Reference` can also be removed; it's part of the `.SchemaMarkdown` output